vendor/CMakeLists.txt

```diff
@@ -61,3 +61,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_FILE_OFFSET_BITS=64")
 # Targets which should be installed by `make install`.
 install(TARGETS adb fastboot "${ANDROID_MKE2FS_NAME}"
 	simg2img img2simg append2simg DESTINATION bin)
+
+# Install license file.
+set(LICENSES "${CMAKE_INSTALL_PREFIX}/share/license/android-tools")
```

Comment:
Wondering if we should use `${CMAKE_INSTALL_DATADIR}/license/android-tools` here :thinking:

See: https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

Reply:
Also decided to use this for the bash completions see d1a9cd6f0ba4d44290a599bf21a592aa3ea098a5 and 45e337f2bf84ceb55c5625aafc5ab5793a91c023